### ISSUE TITLE
New endpoint: SVV (Salzburg, Austria)

### ISSUE THREAD

**Issue by paha77** [public-transport/hafas-client#108]
It seems like that the company Salzburger Verkehrsverbund (https://salzburg-verkehr.at/) is also using HAFAS: https://fahrplan.salzburg-verkehr.at/bin/mgate.exe

I've started creating a new enpoint for it but it seems like their API requires authentication.  Do / must such companies provide access to their API for such open source projects or is it a business decision of their own?  

I don't want to waste time if they don't want their API to be used by others.

Thanks.

**Comment by derhuerst** [comment]
> It seems like that the company Salzburger Verkehrsverbund (https://salzburg-verkehr.at/) is also using HAFAS: https://fahrplan.salzburg-verkehr.at/bin/mgate.exe

Thanks for the hint. [I've added the endpoint to my list](https://gist.github.com/derhuerst/2b7ed83bfa5f115125a5#salzburger-verkehrsverbund).

> I've started creating a new enpoint for it but it seems like their API requires authentication. Do / must such companies provide access to their API for such open source projects or is it a business decision of their own?
>
> I don't want to waste time if they don't want their API to be used by others.

`hafas-client`, as well as its associated libs in other repos, only makes accessing an endpoint *possible*, including credentials you might need. It does *not* try to answer wether accessing that API is allowed, as the answer to this is a) often nuanced and b) unfortunately often defaults to a closed & non-innovative position. Also, I know for fact that `hafas-client` has already helped people access a HAFAS endpoint *legitimately*.

Long story short: Ask the providers of a HAFAS endpoint before querying it in significant amount.

I personally am in favour of adding the SVV profile to this repo. As I personally advocate for public transport data being freely available, I'd default for open source clients & keys, until someone legally demands them to be taken offline.

**Comment by paha77** [comment]
I've already tried to ask for it but the only answer was to open the
startpage of their route planner in my browser. :). Their customer service
had no idea what I was talking about. :(

Thanks.

**Comment by derhuerst** [comment]
If you need help writing an SVV profile and getting the keys, refer to [the guide](https://github.com/public-transport/hafas-client/blob/4/docs/writing-a-profile.md) and/or contact me.

**Comment by paha77** [comment]
Thanks, I wrote you an email. :)

**Comment by derhuerst** [comment]
Because SVV has the HAFAs webapp thing deployed now, the `mic`/`mac` authentication is no longer necessary. See also https://gist.github.com/derhuerst/35808010286e4b6a913306449cd26141.

**Comment by derhuerst** [comment]
`hafas-client@5.5.0` includes the SVV profile from #171. 🎉
